Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM qazbnm456/fbctf_base:latest
```

Comment:
Still true I guess, but this custom image probably has all the packages installed that would be installed by the setup script, so that doesn't have to happen for people using this container image